src/pydash/helpers.py

```diff
@@ -11,15 +11,23 @@
 import pydash as pyd
 from ._compat import Iterable, PY2, iteritems, getfullargspec, string_types
 
+_attr_ignored_types = (dict, list)
+if not PY2:
+    try:
+        from typing import Sequence, Mapping
```

Comment:
Instead of importing from `typing` module (typically that's used for type hinting and not runtime type checking), we can import from `collections.abc` (Python 3) or `collections` (Python 2) and then can use the same set of types for both versions. We can put the conditional import in `pydash._compat` and then import `Sequence` and `Mapping` from there. Then, we'd only need `(Sequence, Mapping)` since `dict/list` instances are types of those.

Reply:
That sounds perfect to me.